### DISCUSSION TITLE
Does not generate webp image

### DISCUSSION THREAD

**Discussion by recoverjp** [ddur/Warp-iMagick#11] [Q&A] [answered]
I have a blog running with WordOps, I installed the warp imagick plugin, but when sending the image it is not generating the webp image. What can it be?

Just an observation: I use the WP offload Media plugin that sends the images to amazon's S3. Could this be the problem?

**Comment by ddur** [comment, marked as answer]
> I have a blog running with WordOps, I installed the warp imagick plugin, but when sending the image it is not generating the webp image. What can it be?
> 
> Just an observation: I use the WP offload Media plugin that sends the images to amazon's S3. Could this be the problem?

Very likely. WebP cloning expects and requires JPEG/PNG on the local server, not in the remote location.

Another reason could be that WP offload does not move WebP to remote location.

Third reason could be that WordOps WebP redirection does apply to local server, not on remote server.

**Reply by recoverjp** [reply]
I am sad to hear this because I really like this plugin, it serves me very well in another project. Thanks for the answer.
